packages/block-editor/src/store/selectors.js

```diff
@@ -192,8 +170,7 @@ export const getBlock = createSelector(
 		};
 	},
 	( state, clientId ) => [
-		...getBlockAttributes.getDependants( state, clientId ),
```

Comment:
Can we safely remove this? I tend to think if there's another selector being called and it has dependants, we should have this sort of reference.

I ask also because there's a few other places in this file we still include `getBlockAttributes.getDependants`, and similarly would we consider to replace those with references to the cache value instead? Unlike the above "guideline", it's not very clear when and why we can reference the cache instead of other dependants values.

Reply:
Yeah, so the cache value is basically saying, the block object need to be refreshed. It includes both the attributes and the inner blocks. 

So if we only depend on the attributes of the block, using the cache key might result in the selector being called again even if not needed. 

I can add a comment to clarify here.